ngraph/core/include/ngraph/op/depth_to_space.hpp

```diff
@@ -65,11 +65,6 @@ namespace ngraph
             protected:
```

Comment:
Please check includes in this file. Looks like #include `ngraph/op/util/fused_op.hpp` shouldn't be needed.

Reply:
OK